layersvt/device_simulation.md

```diff
@@ -95,32 +95,71 @@ JSON file formats consumed by the DevSim layer are specified by one of the JSON
 |:----------:|:-------------:|
 | Vulkan v1.0 | https://schema.khronos.org/vulkan/devsim_1_0_0.json# |
 | Vulkan v1.1 | https://schema.khronos.org/vulkan/devsim_1_1_0.json# |
+| Vulkan v1.2 | https://schema.khronos.org/vulkan/devsim_1_2_0.json# |
 | VK_KHR_portability_subset | https://schema.khronos.org/vulkan/devsim_VK_KHR_portability_subset-provisional-1.json# |
+| VK_KHR_8bit_storage | https://schema.khronos.org/vulkan/devsim_VK_KHR_8bit_storage_1.json# |
 | VK_KHR_16bit_storage | https://schema.khronos.org/vulkan/devsim_VK_KHR_16bit_storage_1.json# |
+| VK_KHR_buffer_device_address | https://schema.khronos.org/vulkan/devsim_VK_KHR_buffer_device_address_1.json# |
+| VK_KHR_depth_stencil_resolve | https://schema.khronos.org/vulkan/devsim_VK_KHR_depth_stencil_resolve_1.json# |
```

Comment:
Isn't VK_KHR_draw_indirect_count missing?

Reply:
https://www.khronos.org/registry/vulkan/specs/1.2-extensions/html/vkspec.html#VK_KHR_draw_indirect_count

"Vulkan 1.2 is supported and this extension is not, the entry points vkCmdDrawIndirectCount and vkCmdDrawIndexedIndirectCount are optional. " 

How this case is actually handled?

Reply:
VK_KHR_draw_indirect_count does not provide any queries Devsim is interested in.

Reply:
VK_KHR_draw_indirect_count is itself a query